### PR TITLE
Bug fix: scroll chat to bottom when it is opened and when it is updat…

### PR DESCRIPTION
**Pull request by Happy0**
…ed on sending or receiving a message.

### CODE REVIEW

**Comment by Happy0** [comment]
I tried it with the keyboard enabled in google chrome developer tools, but I don't know if that's an accurate representation of how it will behave in the app.

I guess I'll try to get it on my phone tomorrow / soon just to make sure.

File: project/src/js/ui/shared/round/chat.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ export function chatCtrl(root, isShadowban) {
     redraw();
   }.bind(this);
 
+  this.scrollChatToBottom = function(el) {
+    el.scrollTop = 999999;
```
Comment (veloce) [inline project/src/js/ui/shared/round/chat.js]: `el.scrollTop = el.scrollHeight` is supposed to work, why did you change it?

**Comment by Happy0** [comment]
It doesn't work for me on android. Maybe you need el.dom.scrollHeight = ?.

el is not a Dom element but has a 'don't property

**Comment by veloce** [comment]
Ah... yes you're right, it's a regression. `el` here is the vnode, not the dom element.
So yes it should be: 

```js
this.scrollChatToBottom = function(el) {
  el.scrollTop = el.scrollHeight;
}
...

oncreate: (vnode) => this.scrollChatToBottom(vnode.dom)
onupdate: (vnode) => this.scrollChatToBottom(vnode.dom)
```

**Comment by Happy0** [comment]
Will do! 'el.scrollHeight' is better than = 9999999 :P.

**Comment by Happy0** [comment]
I made the changes using github's web interface, will test when i get home later.

**Comment by Happy0** [comment]
I've tested this both in chrome dev tools and as an android APK (including with the keyboard open.) Seems to work fine!

Feel free to merge if you're happy with it.
